common/protob/messages.proto

```diff
@@ -256,6 +256,8 @@ enum MessageType {
     MessageType_StellarBumpSequenceOp = 221 [(wire_in) = true];
     MessageType_StellarManageBuyOfferOp = 222 [(wire_in) = true];
     MessageType_StellarPathPaymentStrictSendOp = 223 [(wire_in) = true];
+    // omitted: StellarCreateClaimableBalanceOp is not a supported operation, would be 224
```

Comment:
```suggestion
    reserved 224;  // omitted: StellarCreateClaimableBalanceOp
```
(please make the same change for InflationOp above while you're at it)

Reply:
d702b78afe1d5e6bf470c224102f9d431c742ad0

I also added the test for T1. We can decide whether to keep it after confirmation.